packaging/hudi-flink-bundle/pom.xml

```diff
@@ -633,5 +634,15 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>include-flink-sql-connector-hive</id>
```

Comment:
Can we exclude the conflict dependencies instead of introducing a new profile?

Reply:
> Can we exclude the conflict dependencies instead of introducing a new profile?

We try to exclude the conflict before, but too many conflicts dependencies between flink-sql-connector-hive and our hudi-flink-bundle, we failed to exclude all conflicts.

Reply:
OK~